subsys/bluetooth/controller/ticker/ticker.c

```diff
@@ -1606,6 +1583,10 @@ u32_t ticker_init(u8_t instance_index, u8_t count_node, void *node,
 	instance->count_node = count_node;
 	instance->node = node;
 
+	while (count_node--) {
+		instance->node[count_node].priority = 0;
+	}
```

Comment:
This should be done in `ticker_job_op_start`. If not, the priorities are retained between ticker start/stop sessions, when they are different things.

Reply:
This is intentional. During conn interval updates, tickers are stopped and restarted in ULL. If priority is initialized in the start operation, priority is "suddenly" lost. In my opinion, letting priority stick makes sense because the application should always set priority on a connection anyway (if needed).

Reply:
As me and Morten talked about offline, this priority "spill over" should be fixed. A possible solution would be to introduce a Restart user op - to be used by conn update. This leaves Start to do zero init of priority.

Reply:
> As me and Morten talked about offline, this priority "spill over" should be fixed. A possible solution would be to introduce a Restart user op - to be used by conn update. This leaves Start to do zero init of priority.

We decided to accept it as is, and create an issue on it.